### DISCUSSION TITLE
Problem with VR and planar reflections, need help 😥

### DISCUSSION THREAD

**Discussion by jonasvalentinpersson** [crest-ocean/crest#1144] [Q&A]
Hi!
I have problem getting the planar reflections to work properly. 

Its a VR game using single pass stereo rendering (not instanced)
Im using the Default renderer 4.11 crest version, (To much work to upgrade to later versions and URP, plugins etc).

the planar reflection does not end up ( its moving around) in the right place and is completly different on right and left eye.
Is this fixed in later version? Can I patch it?

![image](https://github.com/wave-harmonic/crest/assets/50102388/9cedf5e8-4d45-4fb6-a07a-7258d9534c25)

**Comment by magenta404** [comment]
Honestly I cannot remember. That stereo mode support was removed ages ago. If I remember correctly that mode was one of the ones that worked with planar reflections. Anyhow, try searching through the release notes.
